samwise.js

```diff
@@ -7,19 +7,18 @@
 /* global clearTimeout, setTimeout, document */
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?